packages/date-helpers/src/index.ts

```diff
@@ -75,12 +76,17 @@ export const relativeClosingDate = ({
   timeZone,
   customFormat,
 }: relativeClosingDateOptions): string => {
+  const inTz = timeZone ? tz(timeZone) : undefined;
   // A date formatting function that can use time zones optionally
-  const myFormatFunc = timeZone
-    ? (date: Date, formatPattern: string, options?: { locale?: Locale }) =>
-        formatInTimeZone(date, timeZone, formatPattern, options)
-    : (date: Date, formatPattern: string, options?: { locale?: Locale }) =>
-        format(date, formatPattern, options);
+  const myFormatFunc = (
```

Comment:
You probably don't need `myFormatFunc` anymore since it is always the same function now.  You can just call `format` directly.

Reply:
Perfect, thanks. I was not 100% sure about removing it so I left it there initially. It has been removed in [remove format func](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11742/commits/b0ba8d4430b7d18910878f1a7cc2fafb99ce91fc)

Reply:
Nice.